static/app/components/onboarding/gettingStartedDoc/utils.tsx

```diff
@@ -101,12 +104,14 @@ export const getReplaySDKSetupSnippet = ({
   Sentry.init({
     dsn: "${dsn}",
 
-    // This sets the sample rate at 10%. You may want this to be 100% while
-    // in development, then sample at a lower rate in production.
-    replaysSessionSampleRate: 0.1,
-    // If the entire session is not sampled, use the below sample rate to sample
-    // sessions when an error occurs.
-    replaysOnErrorSampleRate: 1.0,
-
-    integrations: [new Sentry.Replay()],
+    integrations: [
+      new Sentry.Replay(),
+    ],
+    // Session Replay
+    replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+    replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
   });`;
+
+export const replayJsFrameworkOptions: PlatformIntegration[] = platforms.filter(p =>
```

Comment:
Nit: as it is only used for the replay sidebar and not in the actual "doc files", would it make sense to move it into or close to the sidebar component?